src/Cake.Core.Tests/Unit/IO/NuGet/PackageReferenceBundlerTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Cake.Core.IO;
+using Cake.Core.IO.NuGet;
+using NSubstitute;
+using Xunit;
+
+namespace Cake.Core.Tests.Unit.IO.NuGet
+{
+    public sealed class PackageReferenceBundlerTests
+    {
+        public sealed class TheParseMethod
```

Comment:
Tests do a lot of in the `Given` step. Maybe refactor out to a fixture and add methods to set the fixture in the right state. Would increase readability of the tests.